osu.Game/Overlays/Comments/DrawableComment.cs

```diff
@@ -317,6 +354,30 @@ private void load(OverlayColourProvider colourProvider)
             };
         }
 
+        private void deleteComment()
+        {
+            dialogOverlay.Push(new ConfirmDialog("Do you really want to delete your comment?", () =>
+            {
+                actionsContainer.Hide();
+                actionsLoading.Show();
+                var request = new CommentDeleteRequest(Comment.Id);
+                request.Success += _ =>
```

Comment:
```suggestion
                request.Success += _ => Schedule(() =>
```

For safety I'd schedule this to the local drawable.